capabilities/test-automation.md

```diff
@@ -1,48 +1,78 @@
 
-The Create and Manage Ephemeral Environments practice enhances the Test Automation DORA capability by providing flexible, production-like testing setups that ensure automated tests are reliable and accurate.
+Creating and managing ephemeral environments provides flexible, production-like testing setups that ensure automated tests are reliable and accurate.
 These temporary environments reduce conflicts, simplify debugging, and improve reproducibility.
 Integrated into CI/CD pipelines, they offer continuous, immediate feedback on code changes, promoting early bug detection and robust software quality.
-Using Infrastructure as Code (IaC) for automation optimizes resources and allows for easy scaling of testing environments, supporting effective test automation.
+Using Infrastructure as Code (IaC) for automation optimizes resources and allows for easy scaling of testing environments.
 
 ### [Follow Functional Core Imperative Shell](/practices/follow-functional-core-imperative-shell.md)
 
-The Follow Functional Core, Imperative Shell practice supports the Test Automation capability by facilitating the creation of highly testable and maintainable code.
-The functional core, being side-effect-free, allows for straightforward unit testing, as its pure functions yield predictable results and don't rely on external states.
+Following the Functional Core, Imperative Shell pattern facilitates the creation of highly testable and maintainable code.
+The functional core, being side-effect-free, allows for straightforward unit testing -- its pure functions yield predictable results and don't rely on external states.
 This ensures that the core business logic is thoroughly tested and reliable.
-The imperative shell handles side effects and interactions with external systems, which can be tested through integration tests.
-This clear separation simplifies the testing process, improves test coverage, and provides faster, more reliable feedback during development, which is crucial for robust and efficient test automation.
+The imperative shell handles side effects and interactions with external systems, which can be checked with integration tests.
+This clear separation simplifies the testing process, improves test coverage, and provides faster and more reliable feedback during development. These outcomes are crucial for robust and efficient test automation.
+
+### [Implement Test-Driven Development](/practices/implement-tdd.md)
+
+Implementing Test-Driven Development (TDD) integrates rigorous testing directly into the development process.
+TDD mandates writing tests _before_ coding, ensuring early bug detection and promoting better software design through continuous refactoring.
+Following the red-green-refactor cycle and principles like Transformation Priority Premise and F. R. I. S. T., TDD ensures reliable and efficient tests that align with the Test Pyramid's principles.
```

Comment:
Will target reader know what the "Test Pyramid's principles" are? Or should we quickly define? (e.g., "...that align with the Test Pyramid's principles of ...")

Reply:
Done.